src/basic/iovec-util.h

```diff
@@ -32,7 +32,8 @@ static inline bool IOVEC_INCREMENT(struct iovec *i, size_t n, size_t k) {
         return true;
 }
 
-#define IOVEC_NULL (struct iovec) {}
+#define IOVEC_NULL (const struct iovec) {}
```

Comment:
Let's instead drop this. This is not really useful, in other similar cases we just do the cast in the open and the compiler can optimize the whole thing away.

Reply:
→ https://github.com/systemd/systemd/pull/29679